nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -172,16 +172,17 @@ in
     };
 
     environmentFile = lib.mkOption {
-      type = with lib.types; nullOr path;
-      default = null;
+      type = with lib.types; coercedTo path lib.singleton (listOf path);
+      default = [ ];
       example = "/var/lib/vaultwarden.env";
       description = ''
-        Additional environment file as defined in {manpage}`systemd.exec(5)`.
+        Additional environment file or files as defined in {manpage}`systemd.exec(5)`.
 
         Secrets like {env}`ADMIN_TOKEN` and {env}`SMTP_PASSWORD`
         should be passed to the service without adding them to the world-readable Nix store.
+        NB: If passing a relative path like `./env-file`, the file will be moved to the Nix store. Use [agenix](https://github.com/ryantm/agenix) or [sops-nix](https://github.com/Mic92/sops-nix) if you want to distribute secrets this way.
```

Comment:
This has nothing to do with it being relative but rather with it being unquoted.

Reply:
(Also note, that the sentence only states "relative --> store", which is true. It does not say anything about non-relative paths)

Reply:
Which, for this particular instance, won't end up in store (as I've shown above).

(Yes, I've done a practical test for this claim).